app/javascript/containers/PlayerContainer.js

```diff
@@ -1,13 +1,15 @@
 import React, { Component } from 'react';
 import PlayerTile from '../components/PlayerTile';
-
+import Search from '../components/Search';
 
 class PlayerContainer extends Component {
   constructor(props) {
     super(props);
     this.state = {
-      players: []
+      players: [],
+      all_players: []
```

Comment:
Maybe call one of these `searchedPlayers` or something?  Also, remember camelCase for JS, underscore is for Ruby.